numeros_pares.py

```diff
@@ -1 +1,4 @@
-
+i=2
+while(i<=20):
```

Comment:
Acá recuerda los bloques de código, si te fijas en peso_lunar.py hiciste el print dentro del bucle for, es decir después de los `:` (dos puntos) del for, hay un tab. Ese sería el bloque al que me refiero.

Reply:
Lo que quise decir es que debería de ser así, por ejemplo:

```python
while(i <= 20):
    print(i)
    i = i + 2
```

Notá que el `print` y `i = i + 2` están dentro del `while` en un bloque de código.